src/gmt_grdio.c

```diff
@@ -3000,18 +3000,18 @@ void gmt_grd_flip_vertical (void *gridp, const unsigned n_cols32, const unsigned
 	gmt_M_str_free (tmp);
 }
 
-bool gmtlib_check_url_name (char *fname) {
+bool gmtlib_found_url_for_gdal (char *fname) {
 	/* File names starting as below should not be tested for existence or reading permissions as they
-	   are either meant to be accessed on the fly (http & ftp) or they are compressed. So, if any of
-	   the conditions holds true, returns true. All cases are read via GDAL support or other. */
-	if (gmt_M_file_is_url (fname) ||
+	   are either meant to be read via GDAL.  Regular URLs (e.g., htthp://, ftp:// are read via libcurl. */
+	if (!strncmp(fname,"WCS:", 4)  ||
+		!strncmp(fname,"WMS:", 4)  ||
```

Comment:
test only for /vsi

Reply:
test also for
!strncmp(fname,"WCS:", 4)
!strncmp(fname,"WMS:",4)

Reply:
?? You mean just check for the 4 characters /vsi instead of the list?
And I thought I am checking wor WCS and WMS here.

Reply:
Yes, the 4 /vsi chars. There are lots of vsi drivers now

Reply:
OK, now just checking for /vsi 4 chars